### PR TITLE
Update framework config

### PR DESCRIPTION
**Pull request by StevenRenaux**
Hi,

Related to #18762.
Let me know if I understood right the desired changes. ;)

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1641,11 +1639,13 @@ To see a list of all available storages, run:
 handler_id
 ..........
 
-**type**: ``string`` **default**: ``session.handler.native_file``
+**type**: ``string`` or ``null`` **default**: ``null``
 
 The service id used for session storage. The default value ``'session.handler.native_file'``
```
Comment (nicolas-grekas) [inline reference/configuration/framework.rst]: By default, the session handler configured by php.ini is used, unless option save_path is used, in which case the default is to store sessions using the native file session handler.

File: reference/constraints/Email.rst [hunk 2]
```diff
@@ -133,11 +130,6 @@ The default value used by this option is set in the
 :ref:`framework.validation.email_validation_mode <reference-validation-email_validation_mode>`
 configuration option.
 
-.. deprecated:: 6.2
-
-    The ``loose`` value is deprecated since Symfony 6.2. Starting from
-    Symfony 7.0, the default value of this option will be ``html5``.
-
```
Comment (alexandre-daubois) [inline reference/constraints/Email.rst]: This needs to be reverted, it will be removed in the 7.0 branch

File: reference/constraints/Email.rst [hunk 3]
```diff
@@ -108,9 +108,6 @@ Parameter        Description
 
 This option defines the pattern used to validate the email address. Valid values are:
 
-* ``loose`` uses a simple regular expression (just checks that at least one ``@``
-  character is present, etc.). This validation is too simple and it's recommended
-  to use one of the other modes instead;
```
Comment (alexandre-daubois) [inline reference/constraints/Email.rst]: This needs to stay until 7.0 also :slightly_smiling_face:

File: reference/configuration/framework.rst [hunk 4]
```diff
@@ -1901,7 +1901,7 @@ This option is related to the `session.sid_bits_per_character PHP option`_.
 save_path
 .........
 
-**type**: ``string`` **default**: ``%kernel.cache_dir%/sessions``
+**type**: ``string`` or ``null`` **default**: ``%kernel.cache_dir%/sessions``
```
Comment (alexandre-daubois) [inline reference/configuration/framework.rst]:
```suggestion
**type**: ``string`` | ``null`` **default**: ``%kernel.cache_dir%/sessions``
```

File: reference/configuration/framework.rst [hunk 5]
```diff
@@ -53,13 +53,11 @@ out all the application users.
 handle_all_throwables
 ~~~~~~~~~~~~~~~~~~~~~
 
-**type**: ``boolean`` **default**: ``false``
+**type**: ``boolean`` **default**: ``true``
 
-If set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
+Set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
```
Comment (alexandre-daubois) [inline reference/configuration/framework.rst]:
```suggestion
When at ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
```

Maybe?

File: reference/constraints/Email.rst (hunk 2 above)
Comment (StevenRenaux) [reply reference/constraints/Email.rst]: You right, it was one of my first contribution and I hadn't paid attention about that.

Comment (alexandre-daubois) [reply reference/constraints/Email.rst]: No worries, congrats for this first one! :slightly_smiling_face:

File: reference/configuration/framework.rst [hunk 6]
```diff
@@ -3035,7 +3030,7 @@ php_errors
 log
 ...
 
-**type**: ``boolean|int`` **default**: ``%kernel.debug%``
+**type**: ``boolean|int`` **default**: ``true``
```
Comment (alexandre-daubois) [inline reference/configuration/framework.rst]:
```suggestion
**type**: ``boolean`` | ``int`` **default**: ``true``
```

File: reference/configuration/framework.rst [hunk 7]
```diff
@@ -53,13 +53,11 @@ out all the application users.
 handle_all_throwables
 ~~~~~~~~~~~~~~~~~~~~~
 
-**type**: ``boolean`` **default**: ``false``
+**type**: ``boolean`` **default**: ``true``
 
-If set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
+When at ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
```
Comment (OskarStark) [inline reference/configuration/framework.rst]:
```suggestion
When set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
```

File: reference/configuration/framework.rst [hunk 8]
```diff
@@ -1641,11 +1639,13 @@ To see a list of all available storages, run:
 handler_id
 ..........
 
-**type**: ``string`` **default**: ``session.handler.native_file``
+**type**: ``string`` | ``null`` **default**: ``null``
+
+By default at `null` if `framework.session.save_path` is not set.
```
Comment (OskarStark) [inline reference/configuration/framework.rst]:
```suggestion
By default at ``null`` if ``framework.session.save_path`` is not set.
```

File: reference/configuration/framework.rst [hunk 9]
```diff
@@ -1641,11 +1639,13 @@ To see a list of all available storages, run:
...
+The session handler configured by php.ini is used, unless option
+`framework.session.save_path` is used, in which case the default is to store sessions
```
Comment (OskarStark) [inline reference/configuration/framework.rst]:
```suggestion
``framework.session.save_path`` is used, in which case the default is to store sessions
```

File: reference/configuration/framework.rst [hunk 10]
```diff
@@ -53,13 +53,11 @@ out all the application users.
...
 ~~~~~~~~~~~~~~~~~~~~~
 
-**type**: ``boolean`` **default**: ``false``
+**type**: ``boolean`` **default**: ``true``
 
-If set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
+When set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
 thrown by the application and will turn them into HTTP responses.
 
-Starting from Symfony 7.0, the default value of this option will be ``true``.
```
Comment (xabbuh) [inline reference/configuration/framework.rst:61]: Did you mean to target the `7.0` branch?

Comment (StevenRenaux) [reply reference/configuration/framework.rst:61]: Yeah, and yes my target branch wasn't the right one ;)

**Comment by javiereguiluz** [comment]
@OskarStark I tried to merge this but I faced too many conflicts that I can't solve. Could you give it a try? Thanks!

**Comment by OskarStark** [comment]
Sorry, I wasn't able to merge this PR due to the conflicts. I recreated it by hand in
* https://github.com/symfony/symfony-docs/pull/19360

**Pull request closed without merging**